README.md

```diff
@@ -2,8 +2,10 @@
 
 This project is a Godot-powered game.
 
+## Vision
 
-This will be a 2D Metroidvania style platformer.
-So it will focus on exploring, puzzlesolving and key hunting;
-with some light combat
-Current games similar to it are Hollow Knight and Celeste.
+Arachnivania is a 2D Metroidvania style platformer that will focus on exploring, puzzlesolving and key hunting with some light combat. Current games similar to it are Hollow Knight and Celeste.
+
+## Description
+
+In Arachnivania you play as a spider trying to build its web. In the process you will discover new ways to avoid predators and safe guard your home. This includes displaying warning colors or escaping by using silk.
```

Comment:
Will this be complicated to program? I'm still figuring out how to make a basic platformer level.

Reply:
I just wanted to get the basic structure of the doc complete. I just put something up there. We can definitely go over the detailed description.

Reply:
Revised doc to include updated description and story.